store/resources.js

```diff
@@ -0,0 +1,46 @@
+import storeHelper from '~/helpers/store'
+import knitLogger from '~/config/logger'
+import _ from 'lodash'
+
+// Mutation types
+const types = {
+  INJECT_RESOURCE: 'INJECT_RESOURCE'
+}
+
+// Mutation functions
+const injection = storeHelper.createMutationFn(types.INJECT_RESOURCE)
+
+// Module getters
+export const getters = {
+  articles: state => state.articles
+}
+
+// Module state
+export const state = () => ({
+  articles: {}
+})
+
+export const actions = {
+  injectResource ({ state, commit }, params) {
```

Comment:
injectResource**s**, I think

Reply:
Well, I was thinking about injectResources but then I came with an idea that this action injects only one type of resource per call with the unlimited number of items. Maybe I should change the action name then? What do you think @k911 ?

Reply:
@FieryCod Definitely we should allow to inject only one type of resource at a time, but still I think that injectResources is more descriptive but whatever you decide I'm ok with it

Reply:
@k911 I decided to stick to injectResource. I hope that you don't mind.

Reply:
@FieryCod Yeah, no problem 👍